thunderfish/peakdetection.py

```diff
@@ -2,6 +2,84 @@
 import numpy as np
 
 
+def std_threshold(data, samplerate, win_shift, std_factor=5.):
```

Comment:
maybe rename "std_factor" to "th_factor" to make it consistent for each of the 3 functions ??